### DISCUSSION TITLE
Welcome to YouTube-Thumbnail-Downloader Discussions!

### DISCUSSION THREAD

**Discussion by ankurg132** [semikolan-dev/YouTube-Thumbnail-Downloader#10] [General]
Hey there, just curious if we can implement a Youtube video downloader too with this project.

**Comment by mohittk** [comment]
Hey! bro, just check the deploy, you need to fix it. Its due to the conflict in branch maybe.
If needed help connect me, I can help it out.

**Reply by ankurg132** [reply]
Yes there were some issues due to merge conflicts, it would be great if you can create a new PR fixing those issues. 
Check out the last comment in https://github.com/semikolan-co/YouTube-Thumbnail-Downloader/pull/8
